src/python/bot/untrusted_runner/untrusted.py

```diff
@@ -210,8 +213,13 @@ def _get_tls_cert_and_key():
 
     return cert_contents, key_contents
 
-  return (str(compute_metadata.get('instance/attributes/tls-cert')),
-          str(compute_metadata.get('instance/attributes/tls-key')))
+  # TODO(mbarbella): Remove this after migrating to Python 3. The grpc library
+  # has explicit type checks against str.
+  cert_contents = str(compute_metadata.get('instance/attributes/tls-cert'))
+  cert_contents = future_utils.native(cert_contents).encode()
```

Comment:
Maybe create a helper for

future_utils.native(str(something)).encode()
and use in all places.

Reply:
Done.